modules/doc/content/newsletter/2021_09.md

```diff
@@ -5,3 +5,40 @@
 - In [NestedSolve.md], the method `getState()` was added to get the state of the
   solve. This allows the user to check if the solve converged, or if it did
   converged, by what criteria.
+
+## Compressible Navier-Stokes Finite Volume (CNSFV)
+
+Through the `navier_stokes` module, MOOSE now has the ability to model transonic
+and supersonic flows using either Harten, Lax, van Leer, Contact (HLLC) or
+Kurganov-Tadmor discretization strategies for the advective flux. The
+compressible Euler equations are implemented using the finite volume method. We
+have also implemented a variety of slope limiters, including van Leer, min-mod,
+first and second order upwind, central differencing, and QUICK, that can be used
+when interpolating quantities from cell centroids to faces. We do not recommend
+this compressible fluid flow implementation for low Mach number flows as results
+can be overly diffusive when the solution is not smooth, e.g. in high Rayleigh
+number natural convection simulations.
+
+### libMesh update
+
+We updated libMesh. Here is a summary of the changes:
+
+  - Mimic stl and boost constructors in our allocators
+  - Try changing max order for L2 families
+  - Add LumpedMassMatrix class
+  - Test custom FE reinit
+  - `$LIBMESH_BENCHMARK` selection of benchmarking examples
+  - RB: skip reinit when there are no DOFs on Elem
+  - Command-line override of default Metis-vs-Parmetis
+  - init_names() at more points
+  - Fix FE side unit tests with --enable-complex
+  - Begin contains_vertex_of with node equality tests
+  - chunked_mapvector storage option
+  - Weaken partitioner_test for SFC on 8+ processors
+  - Bugfix for PR libmesh/libmesh#2940
+  - Revert "interior_parent fix on distributed refined multi-elem_dim meshes"
+  - interior_parent fix on distributed refined multi-elem_dim meshes
+  - RB EIM update
+  - Fix for deprecated warning in SLEPc not-quite-3.15
+  - Use pool allocator in mapvector
+  - Check for gdb in command line in print_trace when compiling with no gdb backtrace by default
```

Comment:
This is a repeat of entries in July. Which month is this applicable to?

Reply:
I will fix it.